File: leer.c
// Autora: Alexia Rivera Steinberg <lachicadesistemas@gmail.com>
// Licencia: GNU GPL 3.0 
// Consegui ambos scripts en www.github.com/alexiarstein 
// Repo: Laboratorio Nerd
// ------------------------------------------------------------
#include <stdio.h>

int main(int argc, char *argv[]) {
    if (argc != 2) {
        printf("Uso: %s <texto>\n", argv[0]);
        return 1;
    }

    FILE *fp = fopen(argv[1], "r");
    if (fp == NULL) {
        printf("Error: no puedo leer: '%s'\n", argv[1]);
        return 1;
    }

    int c;
    while ((c = fgetc(fp)) != EOF) {
        putchar(c);
    }

    fclose(fp);
    return 0;
}

File: leer.py
#!/bin/python3
#--------------------------------------------
# Autora: Alexia Rivera Steinberg <lachicadesistemas@gmail.com>
# Para mi laboratorio nerd.
# Licencia: GNU GPL 3.0
# Descargar SOLO desde mi github www.github.com/alexiarstein
# -------------------------------------------

import sys
#Chequeamos si se asigno un archivo por param.
if len(sys.argv) != 2:
    print("Uso: leer.py <texto>")
    sys.exit(1)

# en caso de que si, lo imprimimos en pantalla.
with open(sys.argv[1], 'r') as f:
    print(f.read())
